se2csv.py

```diff
@@ -7,16 +7,24 @@
 import sys
 
 from seDataParams import *
+from seDataDevices import ParseDevice
+from seDataDevices import unwrap_metricsDict
 
 # file parameters
-inFileName = ""
-invFileName = ""
-optFileName = ""
+# inFileName = ""
+# invFileName = ""
+# optFileName = ""
```

Comment:
Commented code removed.  I also eliminated an unused `from seDataParams import *`

I thought about tidying up the unused runtime options from `getopt.getopt(sys.argv[1:], "ad:hi:o:p:e:")` and decided I shouldn't, since it would break the existing "API" for se2csv.

But on reflection, se2csv probably won't work sensibly now unless the p: -> devsFilePrefix  is provided, so maybe something needs to be done anyway?